src/app/pages/participate/participate.component.html

```diff
@@ -37,10 +37,11 @@ <h1>
       attestations in the beacon chain and shard chains.
     </p>
     <div>
-      The easiest way to get Prysm is to use our installation script in our repository: <a href="https://github.com/prysmaticlabs/prysm" target="_blank">github.com/prysmaticlabs/prysm</a>. First, clone our repository and navigate to it. We'll be using an installation script contained in the repository for the next sections.
-      <pre>
-git clone https://github.com/prysmaticlabs/prysm && cd ./prysm
-</pre>
+      The easiest way to get Prysm is to use our installation script in our repository: <a href="https://github.com/prysmaticlabs/prysm" target="_blank">github.com/prysmaticlabs/prysm</a>.
+      First, make sure you <a href="https://git-scm.com/downloads" target="_blank">have git installed</a> and make a folder named prysm.
```

Comment:
Do you need to have git installed then?

Reply:
Mainly for windows users, since having git installed adds a better CLI tool than the normal windows one. Should I move it somewhere else or mention they use the CLI tool git comes with?